lnrpc/rpc.proto

```diff
@@ -1938,6 +1938,23 @@ message PendingChannelsResponse {
 
         /// The balance in satoshis encumbered in this channel
         int64 limbo_balance = 2;
+
+        /**
+        A list of valid commitment transactions. Any of these can confirm at
+        this point.
+        */
+        Commitments commitments = 3;
+    }
+
+    message Commitments {
+        /// Hash of the local version of the commitment tx.
+        string local_commit_hash = 1;
```

Comment:
nit: would `txid` be more accurate than `hash`? IMO the word `_commit_` is also redundant when the encapsulating field is `Commitments`

Reply:
Good suggestion, change made